dev/AnimatedIcon/AnimatedVisuals/Controls_01_Checkmark.idl

```diff
@@ -0,0 +1,30 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//       LottieGen version:
+//           7.0.0-build.57+g2cc4f0c931
+//       
+//       Command:
+//           LottieGen -Language Cppwinrt -InputFile Controls_01_Checkmark.json
+//       
+//       Input file:
+//           Controls_01_Checkmark.json (30311 bytes created 16:32-08:00 Nov 23 2020)
+//       
+//       LottieGen source:
+//           http://aka.ms/Lottie
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace MU_XC_NAMESPACE
```

Comment:
>MU_XC_NAMESPACE [](start = 10, length = 15)

Is this the best namespace for this type? Should it go in a 'primitives' namespace or something like that? It feels wrong for this to go in the main top-level MUXC namespace.